### PR TITLE
Add batch 4

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: snapshot.yaml [hunk 1]
```diff
@@ -1168,3 +1168,23 @@
   sha: f3693cffd03d7006ba8fc6159e7e9bc6dca5f13e
   tag: 3.7.0-1-gf3693cff
   url: https://github.com/CTFd/CTFd
+- id: fUlpISgw
+  sha: d86224760db6745b11ff4428191a39d262d5aafa
+  tag: None
+  url: https://github.com/lightos/credmap
```
Comment (babenek) [inline snapshot.yaml:1174]: MIT License , but without header

File: snapshot.yaml [hunk 2]
```diff
@@ -1168,3 +1168,23 @@
...
+- id: fWpHKmoM
+  sha: 74ab067c5cc3015d1f522e60446d3711846ef781
+  tag: None
+  url: https://github.com/scwuaptx/CTF
```
Comment (babenek) [inline snapshot.yaml:1178]: GNU GENERAL PUBLIC LICENSE
Version 2, June 1991

File: snapshot.yaml [hunk 3]
```diff
@@ -1168,3 +1168,23 @@
...
+- id: CiZjCWx2
+  sha: d30cf565f0ec8d20729c72a80ee9ab7fcb4677a2
+  tag: None
+  url: https://github.com/AdysTech/CredentialManager
```
Comment (babenek) [inline snapshot.yaml:1182]: MIT License

File: snapshot.yaml [hunk 4]
```diff
@@ -1168,3 +1168,23 @@
...
+- id: LykjZWZ+
+  sha: ba7264c4baee703c94017a2b3bc2fcb7327e033e
+  tag: None
+  url: https://github.com/microsoft/Git-Credential-Manager-for-Mac-and-Linux
```
Comment (babenek) [inline snapshot.yaml:1186]: MIT License

File: snapshot.yaml [hunk 5]
```diff
@@ -1168,3 +1168,23 @@
...
+- id: TSF8XXRy
+  sha: c4b093581b696619615574309a9ffb352b11e1a0
+  tag: None
+  url: https://github.com/aws/credentials-fetcher
```
Comment (babenek) [inline snapshot.yaml:1190]: Apache License
                           Version 2.0, January 2004

**Review by babenek — APPROVED** [review]
Approved

**Review by babenek — APPROVED** [review]
Approved

**Review by xDizzix — APPROVED** [review]

**Pull request merged**